src/components/Nav.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import '../Nav.css';
+
+const Nav = () => (
+  <header>
+    <Link to="/" className="text-logo">Math Magicians</Link>
+    <nav>
+      <Link to="/">Home</Link>
+      <Link to="/calculator">Calculator</Link>
+      <Link to="/quote">Quote</Link>
```

Comment:
- [ ] **[OPTIONAL]** You can import `NavLink` component from `react-router-dom` and use it instead of `Link`. This will allow you to have an active nav link and be able to style the active link 😉. Check this [link](https://reactrouter.com/docs/en/v6/api#navlink) to see how to use `NavLink`